content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,360 @@
+=======
+Uruguay
+=======
+
+.. |DGI| replace:: :abbr:`DGI (Dirección General Impositiva)`
+.. |EDI| replace:: :abbr:`EDI (Electronic Data Interchange)`
+.. |CAEs| replace:: :abbr:`CAEs (Constancia de Autorización de Emisión)`
+
+.. _uruguay/intro:
+
+Introduction
+============
+
+With the Uruguayan localization, you can generate electronic documents with its XML, fiscal folio,
+electronic signature and connection to tax authority Dirección General Impositiva (DGI) through
+Uruware.
+
+The supported documents are:
+
+- e-Invoice, e-Invoice Credit Note, e-Invoice Debit Note;
+- e-Ticket, e-Ticket Credit Note, e-Ticket Debit Note;
+- Export e-Invoice, Export e-Invoice Credit Note, Export e-Invoice Debit Note.
```

Comment:
Hello @auva-odoo 

Thanks for the review, but here it actually makes sense to have them capitalized, because each is a type of document and that's the way they are called in the database. This time we did it on purpose 😉. 

Cheers!

Fernando 

@samueljlieber

Reply:
Then in this case we should use the :guilabel: formatting too :slightly_smiling_face:

Reply:
Agreed! 😉